lib/Dancer2/Manual.pod

```diff
@@ -3090,7 +3075,7 @@ Or to request a specific layout, of course:
     };
 
 Some tokens are automatically added to your template (C<perl_version>,
-C<dancer_version>, C<settings>, C<request>, C<params>, C<vars> and, if you
+C<dancer_version>, C<settings>, C<request>, C<query_parameters>, C<body_parameters>, C<route_parameters>, C<vars> and, if you
 have sessions enabled, C<session>). Check L<Dancer2::Core::Role::Template>
```

Comment:
Hmm, I thought I checked this. I will verify and then send another commit as needed.